README.md

```diff
@@ -139,7 +139,11 @@ The [state machine example](examples/state-machine/StateMachine.hs) is a pretty
 ### [Echidna.Solidity](lib/Echidna/Solidity.hs)
 
 This module provides `loadSolidity`, which takes a solidity source file and provides a VM with the first contract therein loaded as well as a `fuzz`-compatible ABI definition.
-At the moment, it is only compatible with single-contract solidity files, though if given a multi-contract file it will load the first and print a warning.
+
+Support for multiple contracts in a single file along with importing files from an unsupported directory has bee added by using the following optional command line arguments:
```

Comment:
Let's restructure this a bit for clarity. In my opinion, this section should just be a description of Echidna.Solidity's API, and then we can move stuff about CLI usage to the "Usage as an Executable" section